pyproject.toml

```diff
@@ -292,7 +292,7 @@ show_contexts = true
 skip_covered = false
 
 [tool.coverage.paths]
-source = ["src", "**/site-packages"]
+source = ["src", "**/site-packages", "tests"]
```

Comment:
This is wrong. It's a path mapping. src!=tests. This is why I didn't call this entry "source" in the original config so it wouldn't be this confusing.